### DISCUSSION TITLE
Add Norwegian Bokmål

### DISCUSSION THREAD

**Discussion by TheNumber100IsTaken** [ScratchAddons/ScratchAddons#2073] [Translation] [answered]
Can you guys add norwegian bokmål?

**Comment by cobaltt7** [comment]
It needs to be translated first
________________________________
From: TheNumber100IsTaken ***@***.***>
Sent: Sunday, April 4, 2021 9:33:20 AM
To: ScratchAddons/ScratchAddons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ScratchAddons/ScratchAddons] Add Norwegian Bokmål (#2073)


Can you guys add norwegian bokmål?

**Reply by TheNumber100IsTaken** [reply]
I mean add Norwegian bokmål to transfiex so i can translate it.

**Comment by pufferfish101007** [comment]
@GrahamSH-LLK

**Comment by GrahamSH-LLK** [comment, marked as answer]
At school, @apple502j needs to approve new languages.

**Reply by TheNumber100IsTaken** [reply]
oh ok
